docs/get-started/getting-started-nextjs.mdx

```diff
@@ -113,9 +113,12 @@ import AddAppAdminConsole from '../includes/_add-application-console.mdx';
   | **Grant Type** | Authorization Code<br /><GrantTypeAuthorizationCode/> |
   | **All other options** | Use the default values for the remaining options |
 
-4. Click the **Authenticator Config** tab and select **Hosted Web** as the Configuration Type.
-
-  It's the simplest type of authenticator configuration, where Beyond Identity handles the passkey creation and selection experience. For more information on authenticator configuration types, see [authenticator configurations](/docs/next/authentication).
```

Comment:
I think this is ok, though I wonder if we should keep the link for more information on authenticator configuration types. Alternatively, we can add a "Why Hosted Web?".

Reply:
A "Why Hosted Web?" sounds like a good idea!